visualization/app/codeCharta/state/store/metricData/nodeMetricData/nodeMetricData.reducer.ts

```diff
@@ -20,12 +20,18 @@ export function nodeMetricData(state = setNodeMetricData().payload, action: Node
 
 function setNewMetricData(fileStates: FileState[], blacklist: BlacklistItem[]) {
 	const hashMap: Map<string, number> = new Map()
+	const secondMap: Map<string, number> = new Map()
```

Comment:
```suggestion
	const metricMaxValues: Map<string, number> = new Map()
	const metricMinValues: Map<string, number> = new Map()
```
Please adjust the other variable names...